server/src/config/config.js

```diff
@@ -33,6 +33,11 @@ class Config {
         }
       ]
     }
+
+    this.altairConfig = {
+      endpointURL: '/graphql',
+      initialQuery: fs.readFileSync(path.resolve(__dirname, './playground.gql'), 'utf8').replace(/\n/g, ' '),
```

Comment:
Replace could fail if file is missing. 
We should also disable playground config itself as having two playgrounds in the single app makes no sense.

Reply:
We were importing from a file right? Taking all the initial queries from a file seems like a good practice.

Reply:
I meant that this file may not exist or be removed. However let's ignore that for the moment.

Reply:
I have made the other changes.